pkg/framework/host.go

```diff
@@ -145,7 +146,13 @@ func (h *Host) InstallAwsOperator(values string) error {
 	if _, err := tmpfile.Write([]byte(awsOperatorChartValuesEnv)); err != nil {
 		return microerror.Mask(err)
 	}
-	if err := runCmd("helm registry install quay.io/giantswarm/aws-operator-chart@1.0.0-${CIRCLE_SHA1} -- -n aws-operator --values " + tmpfile.Name()); err != nil {
+
+	operation := func() error {
+		return HelmCmd("registry install quay.io/giantswarm/aws-operator-chart@1.0.0-${CIRCLE_SHA1} -- -n aws-operator --values " + tmpfile.Name())
+	}
+	notify := NewNotify("aws-operator-chart install")
+	err = backoff.RetryNotify(operation, NewCustomExponentialBackoff(), notify)
```

Comment:
We work towards abstraction and injection. I would like to configure the backoffs. Later we might want to write tests and then we can configure stop backoffs or whatever we need without refactoring it. Lets please keep things configurable.

Reply:
Sorry but i don't understand, could you please elaborate what do you mean by that? You can always pass an ad-hoc `&backoff.ExponentialBackOff` to `backoff.RetryNotify` if you want to customize further, `NewCustomExponentialBackoff` and `NewNotify` just give defaults.

Reply:
We try to improve the framework here. I already did a lot of restructuring and refactoring. The more we abstract and make configurable the more testable becomes the code. I want to get into a situation in which we might be able to write tests for our code. At the moment this is not really possible because we assume a lot based on the global scope and process environment and so on. When something like the backoff and helm client is abstracted and configurable we can easily inject mocks and write unit tests or something. All this is just towards a better design and adding code which assumes more based on the global scope or process environment and hard code stuff is not making it better here.

Reply:
Aha, you meant injecting the backoff, ok done PTAL.

Reply:
Yeah yeah, the usual dependency injection thing we do.